projects/aca-content/src/lib/components/recent-files/recent-files.component.ts

```diff
@@ -54,19 +55,35 @@ import { TranslateModule } from '@ngx-translate/core';
     PageLayoutComponent,
     TranslateModule,
     ToolbarComponent,
+    SearchAiInputContainerComponent,
     EmptyContentComponent,
     DynamicColumnComponent
   ],
   templateUrl: './recent-files.component.html',
-  encapsulation: ViewEncapsulation.None
+  styleUrls: ['./recent-files.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  selector: 'aca-recent-files'
 })
 export class RecentFilesComponent extends PageComponent implements OnInit {
   columns: DocumentListPresetRef[] = [];
 
+  private _searchAiInputState: SearchAiInputState = {
```

Comment:
repetitive code, can be placed in the base class

Reply:
done